.github/workflows/beam_LoadTests_Go_Combine_Dataflow_Batch.yml

```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Noting that the README wasn't updated - @Amar3tto and @andreydevyatkin please make sure this gets added to the batched updates if its not already tracked somewhere. If it is tracked, please let me know and I'll stop paying attention to this in reviews

Reply:
Great, just wanted to make sure this was included/tracking for that had started